.env

```diff
@@ -2,6 +2,8 @@
 ##  run docker-compose config to see the templated variables ##
 POSTGRES_USER=usaspending
 POSTGRES_PASSWORD=usaspender
+POSTGRES_CLIENT_VERSION=10
+POSTGRES_DB_VERSION=10.18
```

Comment:
I believe this file was removed from version control in some work that Keith did. @gpontejos a new `.env.template` file exists for people to create their own `.env` file with. Could you move the change to there and also have it default to Postgres 13 since that is what we are upgrading to.